src/main/java/edu/cornell/kfs/module/purap/document/service/impl/CuB2BShoppingServiceImpl.java

```diff
@@ -59,6 +60,8 @@
 import edu.cornell.kfs.sys.businessobject.FavoriteAccount;
 import edu.cornell.kfs.sys.service.UserFavoriteAccountService;
 
+import static org.kuali.kfs.kns.service.KNSServiceLocator.getDataDictionaryService;
```

Comment:
We rarely use `import static` statements in non-unit-test code. You may want to reference KNSServiceLocator directly on the related line. Alternatively, you could add the DataDictionaryService as a property on this B2B Shopping Service and inject it in the related Spring bean XML.